### DISCUSSION TITLE
Lock screen

### DISCUSSION THREAD

**Discussion by o0steveo0o** [UnchartedBull/OctoDash#1698] [Q&A]
Is there a way to lock the screen?  Kids can't help themselves.

**Comment by UnchartedBull** [comment]
not possible, at least from OctoDash side. I don't know whether Raspbian has a similar functionality

**Reply by pciavald** [reply]
raspbian has screenlock functionality, but i'm unsure how octodash could trigger it. That's a reason why we implemented the custom screensize so we can use a bar with os-related stuff like rebooting, wifi, virtual keyboard, octoprint access, usb ejection
